pkgs/development/tools/parsing/re-flex/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, boost
+, autoconf
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "re-flex";
+  version = "3.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Genivia";
+    repo = "RE-flex";
+    rev = "v${version}";
+    sha256 = "L7p2u0tRfC80FDkf+5aNxcMVWlHgJoTirGMsSeGqv9A=";
+  };
+
+  nativeBuildInputs = [ boost autoconf automake ];
+
+  doCheck = true;
+  doInstallCheck = true;
```

```suggestion
  doCheck = true;
  doInstallCheck = true;
```
These aren't needed, they're enabled by default.

---

Thank you, I have edited.